src/query/functions/aggregation/take.go

```diff
@@ -126,6 +141,7 @@ func (n *takeNode) ProcessBlock(queryCtx *models.QueryContext, ID parser.NodeID,
 		return nil, err
 	}
 
+	instantaneous := queryCtx.Options.Instantaneous
```

Comment:
I was basically reusing the same terminology: https://github.com/m3db/m3/blob/690dcd38c1187d85beae8b3297b70f3b54d947aa/src/query/api/v1/handler/prometheus/native/read_common.go#L81
because this parameter is passed from here.